acceptance-tests/apps/postgresqlapp/internal/app/app.go

```diff
@@ -19,12 +18,12 @@ const (
 )
 
 func App(conn *connector.Connector) http.Handler {
-	r := chi.NewRouter()
-	r.Head("/", aliveness)
-	r.Put("/{schema}", handleCreateSchema(conn))
-	r.Delete("/{schema}", handleDropSchema(conn))
-	r.Put("/{schema}/{key}", handleSet(conn))
-	r.Get("/{schema}/{key}", handleGet(conn))
+	r := http.NewServeMux()
+	r.HandleFunc("GET /aliveness", aliveness)
```

Comment:
The path has changed here. The aim of the aliveness endpoint is so that CloudFoundry knows that the app has started successfully, and I don't think changing the path work.

Reply:
You are right. I changed the path